source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -535,7 +535,16 @@ def _set_specific_params(mod):
                 args['_night_bus_filter_max_factor'] = mod.night_bus_filter_max_factor
             if args.get('_max_additional_connections') is None:
                 args['_max_additional_connections'] = mod.max_additional_connections
-
+            if args.get('min_nb_journeys') is None:
```

Comment:
Min_nb_journeys has a default value of 0 in https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/journey_common.py#L172 so the database value won't be use

Reply:
@kinnou02  min_nb_journeys is not initialized in journey_common.py.

Reply:
Look like I doesn't know how to read ><